src/NuGet.Core/NuGet.Configuration/PackageNamespaces/PackageSourceSection.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.Configuration
+{
+    internal class PackageSourceSection
```

Comment:
You seem to using this type only as a pass through, to convert from the dictionary that package namespaces configuration returns. 

We might need such a type eventually, but for now I think it's best to keep it simple and avoid adding this type altogether.

Reply:
Ok. Removing this type.